hugo/data/you_draw_it_karlsruhe.json

```diff
@@ -0,0 +1,193 @@
+{
+  "chart_id": "you-draw-it-paris",
+  "start_year_chart": 2007,
+  "end_year_chart": 2050,
+  "start_year_draw": 2019,
+  "unit": "tausend Tonnen",
+  "min_value": 0,
+  "max_value": 4000,
+  "value_label_ticks": 6,
+  "year_label_ticks": 8,
+  "values": [
+    {
+      "year": 2007,
+      "value": 3134.42
+    },
+    {
+      "year": 2008,
+      "value": 3018.888
+    },
+    {
+      "year": 2009,
+      "value": 2591.406
+    },
+    {
+      "year": 2010,
+      "value": 2700.356
+    },
+    {
+      "year": 2011,
+      "value": 2629.205
+    },
+    {
+      "year": 2012,
+      "value": 2714.637
+    },
+    {
+      "year": 2013,
+      "value": 2658.536
+    },
+    {
+      "year": 2014,
+      "value": 2596.305
+    },
+    {
+      "year": 2015,
+      "value": 2558.452
+    },
+    {
+      "year": 2015,
+      "value": 2558.452
+    },
+    {
```

Comment:
Woher kommen diese Zahlen für die Jahre nach 2015?
Wenn ihr diese aus den Input-Daten aus den Zeilen 62 bis 64 in [data/karlsruhe.csv](https://github.com/codeformuenster/klimawatch/blob/add_karlsruhe/data/karlsruhe.csv#L62) berechnet habt, so stimmen sie wahrscheinlich nicht. Die Zeilen 62 bis 64 spiegeln die aktuellen offiziellen Ziele für Karlsruhe wieder. Diese sind meiner Einschätzung nach jedoch nicht ausreichend, um die Paris-Ziele zu erreichen.
Wie berechnet ihr denn für andere Städte die Zahlen für die Paris-Ziele für zukünftige Jahre?
Ist es möglich die Paris-Ziel-Daten zusätzlich zu den aktuellen Ziel-Daten der Stadt im `.csv` abzulegen?

Reply:
Siehe Skript, was sich aber inzwischen etwas geändert hat durch die neue you-draw-it Version: https://github.com/codeformuenster/klimawatch/blob/master/generate_plots.py#L176

Die geplanten Ziele der Stadt tauchen im you-draw-it Plot gar nicht auf, sondern erst im Plotly-Detailplot danach.

Berechnung der Parisziele basiert auf dem CO2-Budget laut Pariser Vertrag, hochgerechnet auf EinwohnerInnenzahl der einzelnen Städte: https://klimawatch.codefor.de/paris-limits/
bzw. im Skript v.a. hier: https://github.com/codeformuenster/klimawatch/blob/master/generate_plots.py#L101
Da das berechnet wird (und ich es für sinnvoll halte, die Berechnung für alle Kommunen gleich zu haben), kann man die Paris-Ziele nicht selber eingeben.